web/html/src/manager/virtualization/guests-edit.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+
+const React = require("react");
+const ReactDOM = require("react-dom");
+const {Panel} = require("../../components/panel");
+const MessagesUtils = require("../../components/messages").Utils;
+const Network = require("../../utils/network");
+const {GuestProperties} = require("./GuestProperties");
+
+
+function update(data, setMessage) {
```

Comment:
IMO all the api calls and state should be decoupled and isolated in their own components! Can you please take a look into the way we did that here: https://github.com/renner/uyuni/commit/162057d9f31b78db7f6ced5833a978e0af77517e#diff-20041f7e91b6a047ebef7f75aa7407bcR4

This way you could even remove the state "messages" from the component GuestProperties and manager it inside a component virtualization-guests-api.js

This logic could even be reused between the guests.js file!! Example:

```javascript
   <VirtualizationGuestActionApi action="update">
        ({messages, etc}) => renderThings(messages, etc)
   </VirtualizationGuestActionApi>
```